README.md

```diff
@@ -13,9 +13,30 @@ This agent is powered by the [Jenkins Remoting library](https://github.com/jenki
 
 See [Using Agents](https://www.jenkins.io/doc/book/using/using-agents/) for more info.
 
-## Running
+## Configuring agents with this Container Image
```

Comment:
Hi, @MarkEWaite 

Thank you for your advice and suggestions.
I amended the previous commit, and pushed it again.

Thanks!

Reply:
> I amended the previous commit, and pushed it again

Thanks.  Usually it is better to push additional commits rather than to force push.  When you force push, you risk that comments on previous commits will be hidden.